content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Operator Quick Start Evaluation Install
+description: Instructions to install Istio in a Kubernetes cluster for evaluation.
+weight: 9
+keywords: [kubernetes, operator]
+aliases:
+---
+
+This guide installs Istio using the standalone Istio Operator. The only dependencies
+required are a supported Kubernetes cluster and the `kubectl` command. This
+installation method lets you quickly evaluate Istio in a Kubernetes cluster on
+any platform.
+
+To install Istio for production use, we recommend using the [Helm Installation guide](/docs/setup/install/helm/)
+instead, which is a stable feature.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/#downloading-the-release).
+
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
+
+1. Check the [Requirements for Pods and Services](/docs/setup/additional-setup/requirements/).
+
+## Installation steps
+
+1. Install the Istio Operator using the default profile:
+
+    {{< text bash >}}
+    $ kubectl apply -f https://preliminary.istio.io/operator.yaml
+    {{< /text >}}
+
+1. (Optionally) change profiles from the default profile to one of the following profiles:
+
+{{< tabset cookie-name="profile" >}}
+
+{{< tab name="permissive mutual TLS" cookie-value="permissive" >}}
+When using the [permissive mutual TLS mode](/docs/concepts/security/#permissive-mode), all services accept both plaintext and
+mutual TLS traffic. Clients send plaintext traffic unless configured for
+[mutual TLS migration](/docs/tasks/security/mtls-migration/).
+
+Choose this profile for:
+
+* Clusters with existing applications, or
+* Applications where services with an Istio sidecar need to be able to
+  communicate with other non-Istio Kubernetes services
+
+Run the following command to switch to this profile:
+
+{{< text bash >}}
+$ kubectl apply -f https://preliminary.istio.io/operator-profile-demo.yaml
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< tab name="strict mutual TLS" cookie-value="strict" >}}
+This profile will enforce
+[mutual TLS authentication](/docs/concepts/security/#mutual-tls-authentication) between all clients and servers.
+
+Use this profile only on a fresh Kubernetes cluster where all workloads will be Istio-enabled.
+All newly deployed workloads will have Istio sidecars installed.
```

Comment:
This sentence implies there is something with this profile that enables automatic sidecar injection

Reply:
This was in the original quickstart documentation.  Copied verbatim :) . I think we can clean up the descriptions after we have the workflow and scrpiting nailed down as a seperate PR if you don't object.

Reply:
ah. Yes, lets do separate PR for this